packages/frontend/src/components/wallet-migration/Disable2FA.jsx

```diff
@@ -102,12 +104,32 @@ const Disable2FAModal = ({ handleSetActiveView, onClose }) => {
     },[initialAccountIdOnStart, batchDisableNotStarted]);
 
     useEffect(() => {
+        // checks if current wallet has bricked account and if so handles it
+        const handleDeserializedContractState = async (account) => {
```

Comment:
I think better name for this method is `showBrickedAccountModal` as it does not quite handle the deserialized state and just shows the modal instead.